app/celery/reporting_tasks.py

```diff
@@ -33,13 +28,35 @@ def create_nightly_billing(day_start=None):
     for i in range(0, 4):
         process_day = day_start - timedelta(days=i)
 
-        transit_data = fetch_billing_data_for_day(process_day=process_day)
+        create_nightly_billing_for_day.apply_async(
+            kwargs={'process_day': process_day.isoformat()},
+            queue=QueueNames.REPORTING
+        )
+
+
+@notify_celery.task(name="create-nightly-billing-for-day")
+@statsd(namespace="tasks")
+def create_nightly_billing_for_day(process_day):
+    process_day = datetime.strptime(process_day, "%Y-%m-%d").date()
+
+    start = datetime.utcnow()
+    transit_data = fetch_billing_data_for_day(process_day=process_day)
+    end = datetime.utcnow()
+
+    current_app.logger.info('create-nightly-billing-for-day {} fetched in {} seconds'.format(
```

Comment:
Weird we have this log message - we could just add statsd annotation to the query methods.